user_docs/en/changes.t2t

```diff
@@ -20,6 +20,7 @@ What's New in NVDA
     -
   -
 - Reporting row and column headers is now supported in contenteditable HTML elements. (#14113)
+- Error messages referenced with `aria-errormessage` are now reported in Google Chrome and Firefox. (#8318)

```

Comment:
This will need to be moved to the 2024.3 change log when it is created

```suggestion
- Error messages referenced with ``aria-errormessage`` are now reported in Google Chrome and Firefox. (#8318)
```